docs/sites.yml

```diff
@@ -9069,3 +9069,17 @@
     - Marketing
     - Design
   built_by: João Matos (Global Creative Studio - Education First)
+- title: webman.pro
+  main_url: https://webman.pro/
+  url: https://webman.pro/
+  description: >
+    {titleofthesite} is an awesome portfolio and technical blog where
```

Comment:
This does not work, see https://www.gatsbyjs.org/showcase/mattferderer.com

Reply:
I think the issue caused by unclear [documentation](https://www.gatsbyjs.org/contributing/site-showcase-submissions/). I think, this line confuses people:
```
description: >
    {titleofthesite} is a shiny new website built with Gatsby v2 that makes important contributions towards a faster web for everyone.
```

We definitely have to fix the documentation somehow. Because for me `{titleofthesite}` looks like a variable that replaces on build. But it doesn't.

Reply:
@DanielRuf  How do you think should I create the issue in that case?

Reply:
somehow it makes sense to remove all curly braces in the example to avoid the confusion that this are variables... or make a additional note to replace the values in curly braces...

Reply:
btw: @DanielRuf fixed your example in #20365

Reply:
Thanks for mentioning that @wwwebman I'll take a look at updating that documentation!

Reply:
@laurieontech => #20373